index.html

```diff
@@ -62,7 +62,7 @@ <h3 class="underlined">My Recent Works <span class="underlined-hr"></span> </h3>
         <div class="project-list">
           <div class="project-item">
             <div class="project-item-img">
-          <img src="./images/IMAGE5.PNG.png" alt="" class="img">
+          <img src="./images/IMAGE5.PNG.png" alt="yoga" class="img">

```

Comment:
An alt message is added

Reply:
- Yes, you have done well. But please add a link for each of the changes you have made to the PR report you have created,